automation/group_vars/all/vars.yaml

```diff
@@ -0,0 +1,74 @@
+---
+ansible_python_interpreter: /usr/bin/python3
+
+env_type: minikube #minikube  #openshift
+eda_deploy_operator: "true"
+eda_deploy_server: "true"
+
+eda_namespace: aap-eda
+
+eda_operator:
+  api_version: kustomize.config.k8s.io/v1beta1
+  namespace: aap-eda
+  disable_name_suffix_hash: true
+  secret_generator_name: redhat-operators-pull-secret
+  operator_literal: eda
+  resource_url: github.com/ansible/eda-server-operator/config/default?ref=0.0.11
+  image_name: quay.io/ansible/eda-server-operator
+  image_new_tag: 0.0.11
+
+eda_operator_files_path: roles/eda-deploy-operator/files
+eda_server_files_path:  roles/eda-deploy-server/files
+
+eda_server:
+  pvc_name: eda-postgres-13-volume
+  access_modes: ReadWriteOnce
+  storage_request: 500Mi
+  namespace: aap-eda
+
+  admin_user: admin
+  admin_password_secret: eda-admin-password
+
+  ingress_type: ingress
+  ingress_class_name: nginx
+  hostname: eda.local 
+  
+  automation_server_url: https://awx.local
+  automation_server_ssl_verify: "no"
+ 
+  image: quay.io/ansible/eda-server
+  image_version: sha-98eaec9
+ 
+  image_web: quay.io/ansible/eda-ui
+  image_web_version: 2.4.860
+
+  api:
+    replicas: 1
+
+  ui:
+    replicas: 1
+
+  worker:
+    replicas: 2
+
+  redis:
+    replicas: 1
+
+  database:
+    database_secret: eda-database-configuration
+    #postgres_storage_class: crc-csi-hostpath-provisioner # openshift
+    postgres_storage_class: standard # minishift
+    storage_requirements:
+      requests:
+        storage: 3Gi
+    resource_requirements:
+      requests: {}
+
+eda_server_kustomization:
+  eda_database_configuration_host: eda-postgres-13
+  eda_database_configuration_port: 5432
+  eda_database_configuration_database: eda
+  eda_database_configuration_username: eda
+  eda_database_configuration_password: Ansible123!
+  eda_database_configuration_type: managed
+  eda_admin_password_password: Ansible123!
```

Comment:
Is the double "password" a typo? Also wouldn't it be better to use the same password (testpass) that we use for our docker deployments? https://github.com/ansible/eda-server/blob/85c43985834b0bdcdbe641005c94bff8e1feaf92/scripts/create_superuser.sh#L23

Reply:
@ttuffin sure! I will update it.

Reply:
@ttuffin updated to testpass. About the passwords, eda_database_configuration_password  is used in the secret and the eda_admin_password_password in the configmap.

Reply:
@ttuffin hahaha got it! I was typo  :D thanks.